client/src/index.jsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+const Welcome = () => {
+  return (
+    <div>Hello world</div>
+  );
+};
+
+export default Welcome;
```

Comment:
@janicelam21 
@poema-cavalcanti 
Ditto on the new line here!

Reply:
There are missed new lines throughout--so I'll make this a global comment vs. individual comments.

Reply:
Yes, I was ignoring the red before....but I'll fix it now!